test/attributes.test.ts

```diff
@@ -1,6 +1,7 @@
 import { DataEvent } from "@js-soft/ts-utils";
 import { ConnectorAttribute, ConnectorRelationshipAttribute, ConnectorResponse } from "@nmshd/connector-sdk";
-import { SuccessionEventData } from "@nmshd/runtime";
+import { DeletionStatus, IncomingRequestStatusChangedEvent, ThirdPartyOwnedRelationshipAttributeDeletedByPeerEvent } from "@nmshd/consumption";
```

Comment:
why do you have to import events from consumption?

Reply:
To get type completion in the test event waiter. By default, it will use the event from ts-utils and this has no other property than the namespace.

Reply:
yeah, but events from consumptions should not be used here, instead use the runtime events if **really** necessary.